app/api/sensors/route.js

```diff
@@ -0,0 +1,69 @@
+import { db } from '@/lib/db'
+import { NextResponse } from 'next/server'
+import { S3 } from '@aws-sdk/client-s3'
+import { Upload } from '@aws-sdk/lib-storage'
+
+const uploadFile = async function (file, file_id, sensor_id) {
+  const config = {
+    region: process.env.REGION || 'fr-par',
+    endpoint: process.env.ENDPOINT || 'https://s3.fr-par.scw.cloud',
+    credentials: {
+      accessKeyId: process.env.SCW_ACCESS_KEY_ID,
```

Comment:
I think we should add `SCW_ACCESS_KEY_ID` and `SCW_SECRET_ACCESS_KEY` in the env.example so people can understand they need to set them